data_steward/cdr_cleaner/cleaning_rules/base_cleaning_rule.py

```diff
@@ -125,6 +116,7 @@ class BaseCleaningRule(AbstractBaseCleaningRule):
     string_list = List[str]
     cleaning_class_list = List[AbstractBaseCleaningRule]
     TABLE_COUNT_QUERY = ''' SELECT COALESCE(COUNT(*), 0) AS row_count FROM `{dataset}.{table}` '''
+    affected_tables_list = List[str]
```

Comment:
this should not be a class level variable.  it is still an instance variable.

Reply:
I made the change but I couldn't figure out how it can be used in __init__  function.